src/biz/storage.go

```diff
@@ -21,7 +21,7 @@ type DeviceMapperStats struct {
 
 // ParseDeviceMapperStats parses the output from the info DriverStatus info Array. This information is not consider stable by
 // Docker and could change anytime according to API the docs.
-func ParseDeviceMapperStats(info types.Info) (*DeviceMapperStats, error) {
+func ParseDeviceMapperStats(info system.Info) (*DeviceMapperStats, error) {
```

Comment:
Why this?

Reply:
the linter failed since the type is marked as deprecated